platform/platform-session-invalidator/src/main/java/org/codice/ddf/platform/session/api/HttpSessionInvalidator.java

```diff
@@ -13,8 +13,8 @@
  */
 package org.codice.ddf.platform.session.api;
 
+import java.util.Map;
 import java.util.function.Function;
-import javax.servlet.http.HttpSession;
```

Comment:
❓ Can we remove this bundles dependency on the javax.servlet-api since this was removed?